src/test/java/hudson/plugins/git/extensions/impl/PreBuildMergeTest.java

```diff
@@ -66,7 +68,11 @@ public void testFailedMerge() throws Exception {
         assertTrue("SCM polling should detect changes", project.poll(listener).hasChanges());
 
         FreeStyleBuild secondBuild = build(project, Result.FAILURE);
-        assertThat(secondBuild.getLog(), containsString("Automatic merge failed; fix conflicts and then commit the result."));
+        Locale currentLocale = Locale.getDefault();
```

Comment:
From JavaDoc:
```
Gets the current value of the default locale for this instance of the Java Virtual Machine.
```
I don't know why, but the JRE locale is on my machine English (maybe because I'm using IntelliJ in English?). 
So the test still fails. Git is using the Locale of the OS.

Reply:
Thanks for checking!  That assertion is not valuable enough to retain.  Later assertions handle the check well enough.

Reply:
I've removed the assertion and force pushed the change.